packages/grid/src/GridUtils.js

```diff
@@ -231,11 +231,20 @@ class GridUtils {
     );
   }
 
-  // Returns the column index if the x/y coordinates provided are close enough to the separator, otherwise false
-  static getColumnSeparatorIndex(x, y, metrics, theme) {
+  /**
+   * Gets the column index if the x/y coordinated provided are close enough
+   * @param {Number} x Mouse x coordinate
+   * @param {Number} y Mouse y coordinate
+   * @param {GridMetrics} metrics The GridMetricCalculator metrics
+   * @param {GridTheme} theme The grid theme with potantial user overrides
+   * @param {Number} floatingColumnsWidth The total width of the frozen columns
```

Comment:
As mentioned in another comment, don't have `floatingColumnsWidth` as a param here - the floating columns are already in the metrics, and you should technically handle left/right floating columns.

It might be easiest to test left/right floating columns in the styleguide, modifying one of the mocks there (or making your own). http://localhost:4000/styleguide